.travis.yml

```diff
@@ -17,3 +16,8 @@ addons:
   apt:
     packages:
       - libnetcdf-dev
+      - libfreetype6-dev
+      - libfontconfig1-dev
```

Comment:
Do these changes help in passing travis build?

Reply:
Nope.

Reply:
I will remove them from travs file